src/iotjs_module_buffer.cpp

```diff
@@ -216,6 +235,10 @@ JHANDLER_FUNCTION(Write) {
 
   BufferWrap* buffer_wrap = BufferWrap::FromJBufferBuiltin(*jbuiltin);
 
+  offset = boundRange(offset, 0, buffer_wrap->length());
```

Comment:
The question was for every call of BoundRange. I'm worrying about the following corner case. If we have an example string "abcd" and call BoundRange like this

`BoundRande(4, 0, my_string->length());`

This will return 4, but my_String[4] is out of range. Did I missed something?